js/models/conversations.js

```diff
@@ -815,28 +815,34 @@
 
     notify: function(message) {
```

Comment:
You are correct. This is usually called from `handleDataMessage()` which currently does nothing with that promise. I spent some time with the method as part of my investigation into errant notifications, and thought I'd leave it better than I found it. :0)